test/functional/webauthn_credentials_controller_test.rb

```diff
@@ -212,5 +274,75 @@ class WebauthnCredentialsControllerTest < ActionController::TestCase
     end
 
     should redirect_to :edit_settings
+
+    context "when the user has no other webauthn credentials" do
```

Comment:
The test cases seems a bit wonky here. The structure currently is
```
context "#destroy"
  setup do
    call destroy
  end
   ...

  shoulds for successful codepath
  
  context "when the user has no other webauthn credentials"
        setup do
            call destory
        end
    end

   ... more contexts
end
```

What is happening is that for "when the user has no other webauthn credentials" and other contexts, we're calling destroy twice, once in the setup for  `context "#destroy"` and one for the context in ` "when the user has no other webauthn credentials"`.  This is unnecessary and could lead to unexpected behaviour.

The first setup + shoulds test the one credential case so we can merge that and "when the user has no other webauthn credentials" together into one context.